chassis.pp

```diff
@@ -16,5 +16,10 @@
   source => "/etc/ssl/certs/${fqdn}.crt",
   mode => '0644',
 }
+file { "/vagrant/${fqdn}.key":
```

Comment:
Might need to `require` the file or add the `->` delimiter as on line 14 to ensure the file is actually present. This would have worked from an already provisioned machine but not from scratch I think.

Reply:
Nice catch! #til about the ordering arrow!